services/mozilla-observatory/mozilla-observatory.service.js

```diff
@@ -74,6 +74,7 @@ module.exports = class MozillaObservatory extends BaseJsonService {
           grade: 'A+',
           score: 115,
         }),
+        queryParams: { publish: null },
```

Comment:
Is this something we should start doing for all examples that have query params?